hack/setup-test-cluster.sh

```diff
@@ -5,24 +5,45 @@ set -e
 # expect oc to be in PATH by default
 export OC_TOOL="${OC_TOOL:-oc}"
 
-# Label 1 worker node as worker-rt
-echo "[INFO]: Labeling 1 worker node with worker-rt"
-node=$(${OC_TOOL} get nodes --selector='node-role.kubernetes.io/worker' -o name | head -1)
-${OC_TOOL} label $node node-role.kubernetes.io/worker-rt=""
-
-# Label 2 worker node as worker-sctp
-echo "[INFO]: Labeling 2 worker node with worker-sctp"
-nodes=$(${OC_TOOL} get nodes --selector='node-role.kubernetes.io/worker' -o name | sed -n 2,3p)
+# Label 2 worker nodes as worker-cnf
+echo "[INFO]: Labeling 2 worker nodes with worker-cnf"
```

Comment:
maybe it's worth validating we find enough nodes. I'm not 100% sure the logic i have below is right, but it seems like something like this would work. 

```
if ${#nodes[@]} -ne 2; then
    echo "[ERROR]: CNF test suite requires 2 worker nodes, ${#nodes[@]} worker nodes found."
fi
```

Reply:
Sctp will work with one node only. The connectivity tests will happen on the single node isntead of two different ones. 
The sed expression will return only the first result if we only have one row.